Easydict/NewApp/View/SettingView/Tabs/ServiceTab.swift

```diff
@@ -34,23 +34,23 @@ struct ServiceTab: View {
     }
 
     var serviceList: some View {
-        List {
-            ForEach(Array(serviceTypes.enumerated()), id: \.offset) { index, _ in
+        List(selection: $selectedIndex) {
+            ForEach(Array(zip(serviceTypes.indices, serviceTypes)), id: \.0) { index, _ in
                 ServiceItemView(
                     service: $services[index]
                 ) { isEnable in
                     serviceToggled(index: index, isEnable: isEnable)
                 }
                 .tag(index)
                 .contentShape(Rectangle())
-                .onTapGesture {
-                    if selectedIndex == nil || selectedIndex != index {
-                        selectedIndex = index
-                    } else {
-                        selectedIndex = nil
-                    }
-                }
-                .listRowBackground(selectedIndex == index ? Color("service_cell_highlight") : Color.clear)
+//                .onTapGesture {
+//                    if selectedIndex == nil || selectedIndex != index {
+//                        selectedIndex = index
+//                    } else {
+//                        selectedIndex = nil
+//                    }
+//                }
+//                .listRowBackground(selectedIndex == index ? Color("service_cell_highlight") : Color.clear)
```

Comment:
So, setting selection row highlighting will cause `onMove` to fail? Are there any other good solutions?

Reply:
I tried this code https://stackoverflow.com/a/64194868/8378840  and it seems to work. 

It's not implemented in a very elegant way, but that should be a SwiftUI issue.